src/include/miopen/conv/heur/heur.hpp

```diff
@@ -78,7 +78,7 @@ struct ConvHeur
         return true;
     }
 
-    std::vector<uint64_t>
+    std::vector<std::pair<uint64_t, float>>
```

Comment:
Please define simple struct consisting of Solver ID and AI Heuristic Index and use it instead of std::pair.

Reply:
This is not necessary anymore, not using pair.